Server/tests/test_core_infrastructure_characterization.py

```
@@ -0,0 +1,1334 @@
+            mock_config = MagicMock()
+            mock_config.uuid_file = data_path / "customer_uuid.txt"
+            mock_config.milestones_file = data_path / "milestones.json"
+            mock_config_cls.return_value = mock_config
+
+            collector = TelemetryCollector()
+
+            assert collector._worker is not None
+            assert isinstance(collector._worker, threading.Thread)
+            assert collector._worker.daemon is True
+
+    def test_telemetry_collector_records_event(self, mock_telemetry_config, temp_telemetry_data):
```

Ruff (0.14.14)

Unused method argument: `mock_telemetry_config`

(ARG002)